airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/publish/context.py

```diff
@@ -48,6 +48,7 @@ def __init__(
         use_local_cdk: bool = False,
         python_registry_token: Optional[str] = None,
         python_registry_url: Optional[str] = None,
+        python_registry_check_url: Optional[str] = None,
```

Comment:
I might repeat the same question I asked yesterday, and sorry for that :) : 
I'm not sure `python_registry_url` and `python_registry_check_url` should be nullable as we're always building contexts with them set to hardcoded defaults. 
The `is_package_published` function expects these attributes to be set.
If I'm understanding the code path correctly the thing which guarantees these are set when executing `is_package_published` is this [if statement](https://github.com/airbytehq/airbyte/blob/main/airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/publish/pipeline.py#L355). Should we check that the check url is not null there too?

Reply:
you are right, as they have a default there is no good reason to make them nullable, I will remove that.